### PR TITLE
card element updated

### PR DESCRIPTION
**Pull request by JamesDean8917**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
         fork box
     </div>
     <div class="card">
-        card element added
+        card element updated
```
Comment (JamesDean7) [inline index.html:27]: Can I ask why this part has been changed?

Comment (JamesDean7) [inline index.html:27]: Can I ask why this part has been changed?

Comment (JamesDean7) [inline index.html:27]: Can I ask why this part has been changed?

**Review by JamesDean7 — APPROVED** [review]
Everything seems fine

File: index.html (hunk 1 above)
Comment (JamesDean8917) [reply index.html:27]: Sure

**Pull request merged**